projects/plugins/jetpack/_inc/client/components/navigation/index.jsx

```diff
@@ -104,7 +105,16 @@ export class Navigation extends React.Component {
 							onClick={ this.trackRecommendationsClick }
 							selected={ this.props.location.pathname.startsWith( '/recommendations' ) }
 						>
-							{ _x( 'Recommendations', 'Navigation item.', 'jetpack' ) }
+							{ _x( 'Recommendations ', 'Navigation item.', 'jetpack' ) }
+							{ this.props.newRecommendationsCount > 0 && (
+								<span
+									className={
+										'dops-section-nav-tab__update-badge count-' + this.props.newRecommendationsCount
+									}
+								>
+									<span className="update-count">{ this.props.newRecommendationsCount }</span>
+								</span>
+							) }
```

Comment:
I wonder if it may make more sense to make that a new translatable string with a placeholder, using `createInterpolateElement`. I think in some languages (RTL), they may expect the bubble on the other side of the word.